tests/qos/qos_sai_base.py

```diff
@@ -242,7 +243,7 @@ def __getBufferProfile(self, request, dut_asic, os_version, table, port, priorit
             )[0].encode("utf-8").translate(None, "[]")
         else:
             bufferProfileName = bufkeystr + dut_asic.run_redis_cmd(
-                argv = ["redis-cli", "-n", db, "HGET", keystr, "profile"])[0]
+                argv=["redis-cli", "-n", db, "HGET", keystr, "profile"])[0]
```

Comment:
why no space here while on all other places we do have ' = '?

Reply:
Hi Liat, this is the default value defination of the function argument, no space is required here. Others are the variable assignments, spaces are required.